README.md

```diff
@@ -15,225 +13,24 @@ OpenAPI 3 CLI toolbox with rich validation and bundling features.
 Currently, @redocly/openapi-cli supports these features:
 
 - [x] Multifile validation. No need to bundle your file before validation.
-- [x] Configurable message levels for each rule. You can tailor your experience with @redocly/openapi-cli as you wish.
-- [x] Lightning-fast validation. Check 1 Mb file in less than one second.
-- [x] Human-readable error messages. Now with stacktrace and codeframes.
+- [x] Lightning-fast validation. Lint a 1 Mb file in less than one second.
+- [x] Built-in rules for common validations.
+- [x] Configurable severity levels for each rule.
+- [x] Human-readable error messages with codeframes and stylish format options.
 - [x] Intuitive suggestions for misspelled types or references.
-- [x] Easy to implement custom rules. Need something? Ask us or do it yourself.
+- [x] Easy to implement custom rules. 
```

Comment:
link to custom rules docs?

Reply:
There is a link to the docs below. When we have solid custom rules docs, we can add a deep link here to those.

Reply:
Good